### ISSUE TITLE
Union type require spaces ?

### ISSUE THREAD

**Issue by Erwane** [cakephp/cakephp-codesniffer#361]
Tested with 4.5 and 5.x-dev on cakephp4.4 project with php8.1

![image](https://user-images.githubusercontent.com/712604/173588066-48dc2ff2-b04e-4905-89a8-26b2ed310b7e.png)

**Comment by dereuromark** [comment]
I am sure the sniffer hasnt been adjusted for new php 8 stuff
Maybe you can make a PR to provide those?

Otherwise please use verbose mode and also output here the sniffs that cause this
So we can look into fixing those from our side.

**Comment by Erwane** [comment]
I don't understand :/

```
php8.1 vendor/bin/phpcs -v src/Server/Server.php --no-cache
Registering sniffs in the CakephpDev standard... DONE (136 sniffs registered)
Creating file list... DONE (1 files in queue)
Changing into directory /home/erwane/projects/interfaces/src/Server
Processing Server.php [PHP => 2151 tokens in 337 lines]... DONE in 79ms (0 errors, 0 warnings)
```
command line don't report the error but phpcs in phpstorm yes :/

**Comment by Erwane** [comment]
My bad :/
My phpstorm phpcs inspection was set to `Custom` and point to a file outside of project.
Sorry.
